src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -100,6 +100,7 @@ trait Backend {
     // Sometimes the class name is `anon$1`.  In cases like that, don't print it in the log because it's not adding value
     val cls = this.getClass.getSimpleName
     val clsString = if (cls.startsWith("anon")) "" else s"$cls "
-    s"$clsString[UUID(${backendCall.workflowDescriptor.shortId}):${backendCall.call.name}]"
+    val shard = backendCall.key.index.map(x => s":$x").getOrElse("")
```

Comment:
seems like a good method to add to the implicit class that's already kicking around for these indexes